### PR TITLE
Introduce Query Client Class

### LINKED ISSUE #51: Add python class for query clients
It should be possible to implement a python query client without manually sending any requests. We should provide a `QueryClient` class that provides methods such as `send_query` which query clients can use.

Related to #50.

### PR DESCRIPTION
**Pull request by patrickab**
Closes #51

### CODE REVIEW

File: prefq/examples/imitation_preference_comparisons.py [hunk 1]
```diff
@@ -34,6 +31,8 @@
 from stable_baselines3 import PPO
 from stable_baselines3.common.evaluation import evaluate_policy
 
+from prefq.examples.queryclient import QueryClient
```
Comment (timokau) [inline prefq/examples/imitation_preference_comparisons.py:34]: I think the `queryclient` module should not be part of `examples`. I'd also prefer to use an underscore in the name: I think  `prefq.query_client` would be good.

File: prefq/examples/queryclient.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            )
+            if 200 <= response.status_code < 400:
+                print(f"Query Client: Payload transferred   Query ID: {query_id}")
+
+        except requests.exceptions.RequestException as exception:
+            print("Query Client: Error: send_videos()")
+            print(f"    Exception: {exception}\n")
+
+    def request_feedback(self):
+        """
```
Comment (timokau) [inline prefq/examples/queryclient.py:58]: Please reformat this to adhere to the "single-line summary first, then details" style of docstring (similar to a commit message).

**Review by timokau — CHANGES_REQUESTED** [review]
Looks good! I just have two minor suggestions.

**Comment by patrickab** [comment]
ready to be reviewed

**Review by timokau — APPROVED** [review]
Thank you!

**Pull request merged**